src/source_loc.rs

```diff
@@ -21,12 +26,31 @@ pub struct SourceLoc {
     #[structopt(long = "rev", default_value = "master")]
     pub rev: String,
 
+    /// git user
+    #[structopt(short = "u", long = "user")]
+    pub usr: Option<String>,
+    /// git password
+    #[structopt(short = "p", long = "password")]
+    pub pwd: Option<String>,
+
     /// path of the folder under the source uri to use for template
     #[structopt(long = "source-subfolder", parse(from_os_str))]
     pub subfolder: Option<PathBuf>,
+
+    /// use for self-signed certificate
+    #[structopt(short = "k")]
+    pub unsecure_certificate: bool,
```

Comment:
also add the long value `insecure` like curl, and align name of the field with the long name.